issuuservice-lib/lib/class.issuudocument.php

```diff
@@ -224,56 +322,16 @@ protected function clearObjectXML($document)
     */
     protected function clearObjectJson($document)
     {
-        $doc = new stdClass();
-
-        $doc->username = $this->validFieldJson($document, 'username');
-        $doc->name = $this->validFieldJson($document, 'name');
-        $doc->documentId = $this->validFieldJson($document, 'documentId');
```

Comment:
Do we have a `documentId` equivalent value? This information is required and if we don't have an equivalent value, maybe I have to change this in most parts of plugin.